src/Batch/FileConventions/Src/AzureBatchFileConventions/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Azure Batch File Conventions")]
 [assembly: AssemblyDescription("A convention-based library for saving and retrieving Azure Batch task output files.")]
 
-[assembly: AssemblyVersion("1.0.0.0")] 
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.1.0")] 
```

Comment:
This is our general guidance for versioning.  Using AssemblyVersion = 'Major.0.0.0', we allow additive updates of the assembly without recompiling or requiring an assembly redirect for programs that use this assembly.  It is useful for scenarios like PowerShell or other library usages, where a config file with assembly redirects is not practical.